pxf/pxf-api/src/main/java/org/apache/hawq/pxf/api/FilterParser.java

```diff
@@ -30,26 +34,40 @@
  * interface with two pop-ed operands.
  * <br>
```

Comment:
Can you add this 
The filter string is of the pattern `<attcode><attnum><constcode><constval><constsizecode><constsize><constdata><constvalue><opercode><opernum>`